app/controllers/feedback_controller.rb

```diff
@@ -1,16 +1,16 @@
-class DevContactsController < ApplicationController
+class FeedbackController < ApplicationController
   skip_before_action :authenticate_user!
   skip_verify_authorized only: %i[new create]
+  layout :set_layout, only: %i[new create]
 
   def new
     @contact = Contact.new
   end
 
   def create
     @contact = Contact.new(contact_params)
-
     if @contact.valid?
-      DevContactsMailer.with(contact_params).send_message.deliver_later
+      FeedbackMailer.with(contact_params).send_message.deliver_later
       redirect_to root_path, notice: I18n.t("contacts.create.success")
```

Comment:
```suggestion
      redirect_back_or_to root_path, notice: I18n.t("contacts.create.success")
```

Reply:
Yes - redirect to dashboard would be preferred!